### PR TITLE
Added comments in loadbalancer/fadein_test.go

### PR DESCRIPTION
**Pull request by RomanZavodskikh**
Updates #2346

### CODE REVIEW

File: loadbalancer/fadein_test.go [hunk 1]
```diff
@@ -92,8 +95,9 @@ func testFadeIn(
 			}
 		}()
 
+		// Split fade-in period into buckets and count how many times each endpoint was selected.
 		t.Log("test done", time.Now())
-		t.Log("CSV " + strings.Join(eps, ","))
+		t.Log("CSV timestamp," + strings.Join(eps, ","))
```
Comment (szuecs) [inline loadbalancer/fadein_test.go:100]: here you likely need to change also:
[~/go/src/github.com/zalando/skipper]% ls skptesting/*fadein*
skptesting/analyze_fadein.r*  skptesting/run_fadein_test.sh*

File: loadbalancer/fadein_test.go [hunk 2]
```diff
@@ -130,18 +134,20 @@ func testFadeIn(
...
+		for i, bucketStats := range allBuckets {
 			var showStats []string
 			for _, epi := range eps {
 				showStats = append(showStats, fmt.Sprintf("%d", bucketStats[epi]))
 			}
 
-			t.Log("CSV " + strings.Join(showStats, ","))
+			// Print CSV-like output for, where row number represents time and
+			// column represents endpoint.
+			t.Log("CSV " + fmt.Sprintf("%d,", i) + strings.Join(showStats, ","))
```
Comment (szuecs) [inline loadbalancer/fadein_test.go:150]: see also https://github.com/zalando/skipper/pull/2376/files#r1221462922

**Comment by szuecs** [comment]
:+1:

**Comment by RomanZavodskikh** [comment]
:+1:

**Pull request merged**